tests/unit/utils/Utils.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __UTILS_H__
+#define __UTILS_H__
+
+/*
```

Comment:
The most suitable methods that I've found are ```mkstemp```, ```mkdtemp``` and ```tmpnam```. Unfortunately, none of them can be used because:
* ```mkstemp``` creates a new random file and return a file descriptor to it. It is safe, but some unit tests need only to have the filename to open a new file;
* ```mkdtemp``` creates a random directory and it is not what we need;
* ```tempnam``` does create a random filename, but it suffers from the same problems of ```mktemp``` (http://en.cppreference.com/w/cpp/io/c/tmpnam);

Although some of the classes used inside the unit tests (CSerializableAsciiFile, CSerializableXMLFile and CCSVFile) permits to create a file given its FD, the CStreamingAsciiFile class has not this ability. So, I opted to simply write this small function to resolve the warnings, without touching too much the unit tests code.